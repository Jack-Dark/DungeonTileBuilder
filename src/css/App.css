.absolute {
  position: absolute;
}
.relative {
  position: relative;
}
.w-100 {
  width: 100%;
}
.h-100 {
  height: 100%;
}
.top-0 {
  top: 0;
}
.right-0 {
  right: 0;
}
.bottom-0 {
  bottom: 0;
}
.left-0 {
  left: 0;
}
.z-1 {
  z-index: 1;
}
.z-2 {
  z-index: 2;
}
.flex {
  display: flex;
}
.flex-wrap {
  flex-wrap: wrap;
}
.items-center {
  align-items: center;
}
.justify-center {
  justify-content: center;
}
.justify-between {
  justify-content: space-between;
}
.pa1 {
  padding: 0.25rem;
}
.pa2 {
  padding: 0.5rem;
}
.pa3 {
  padding: 1rem;
}
.fl {
  float: left;
}
.tc {
  text-align: center;
}
.pointer {
  cursor: pointer;
}
.visible-on-parent-hover {
  visibility: hidden;
}
*:hover > .visible-on-parent-hover {
  visibility: visible;
}
