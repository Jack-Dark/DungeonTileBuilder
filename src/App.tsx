import React from "react";
// import logo from "./logo.svg";
import Tile from "./components/Tile";
import "./App.css";

const App: React.FC = () => {
  return <Tile label="foo" value={1} />;
};

export default App;
