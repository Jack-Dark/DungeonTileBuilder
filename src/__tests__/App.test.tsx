import React from "react";
import App from "../App";
import { shallow, mount, render, ShallowWrapper } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

let Dungeon: ShallowWrapper;
describe("The Dungeon Tile Builder app", () => {
  beforeEach(() => {
    const props = {};
    Dungeon = shallow(<App {...props} />);
  });
  it("should match the snapshot", () => {
    expect(Dungeon).toMatchSnapshot();
  });
});
