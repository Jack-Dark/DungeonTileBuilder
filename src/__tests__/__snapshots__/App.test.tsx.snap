// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Dungeon Tile Builder app should match the snapshot 1`] = `
<Tile
  label="foo"
  value={1}
/>
`;
